packages/typst.node/src/compiler/boxed.rs

```diff
@@ -83,16 +91,12 @@ impl BoxedCompiler {
                 }
 
                 let fp = std::path::Path::new(main_file_path.as_str());
-                Some(match world.workspace_root() {
+                Some(match universe.workspace_root() {
                     Some(root) => {
-                        if let Ok(p) = root.strip_prefix(fp) {
```

Comment:
Originallly, `root.strip_prefix(fp)` had its order of operands reversed.

Reply:
We may use `world.entry_state().try_select_path_in_workspace(fp, true)`.